lib/build-cloud.rb

```diff
@@ -220,16 +223,31 @@ def recursive_interpolate_config(h)
         when String
 
             while h =~ /%\{(.+?)\}/
```

Comment:
Probably able to leave them out, I just have a habit of adding them when I'm testing regexs :)